arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -748,7 +743,7 @@ public void tracePostExecution(MessageFrame frame, Operation.OperationResult ope
 
     defers.resolvePostExecution(this, frame, operationResult);
 
-    if (!this.currentFrame().opCode().isCall() && !this.currentFrame().opCode().isCreate()) {
+    if (isExceptional() || !opCode().isCallOrCreate()) {
```

Comment:
This was a bug: exceptional CALL's or CREATE's wouldn't unlatch the stack. The stack would then carry the default value of `0x1337deadbeef`.

Reply:
So the aborting CALL's don't pose an issue AFAICT. I don't know what will happen with
- aborting CREATE's
- failure condition CREATE's